pages/docs/tutorials/validate-documents.md

```diff
@@ -0,0 +1,115 @@
+---
+title: "Studio AsyncAPI document validation"
+description: This tutorial will teach us how to validate AsyncAPI documents using the AsyncAPI Studio tool.
+weight: 120
+---
+
+# Introduction
+This tutorial will teach you how to validate AsyncAPI documents using the [AsyncAPI Studio tool](https://studio.asyncapi.com/).
+
+We will start with a broken AsyncAPI document and troubleshoot via console errors step-by-step until we end up with a valid AsyncAPI document. This process will illustrate how to identify [`REQUIRED` properties in AsyncAPI documents](https://www.asyncapi.com/docs/reference/specification/v2.5.0#A2SObject).
+
+# Background context
+An AsyncAPI document is a file that defines and annotates the different components of a specific Event-Driven API. The format of the file must be JSON or YAML. You can use this document to generate both documentation and code.
+
+The AsyncAPI Studio tool allows you to develop an AsyncAPI document, validate it, preview it, convert it to the latest version, and visualize event flows.
+
+# Copy invalid AsyncAPI document
+Let's pretend we have an invalid AsyncAPI document.
+
+1. Open [Studio](https://studio.asyncapi.com/).
+
+2. Navigate to the left side bar and click on **new file**.
+
+3. Copy and paste the below invalid AsyncAPI document:
+
+```yaml
```

Comment:
example should be based on the same file as the other tutorials.

also I think we should be consistent with other tutorials.

so say: "yo, if you go through this tutorial without performing previous steps, then get an asyncapi file with `asyncapi new --example=tutorial.yml --no-tty`. Paste it in Studio, all works perfect, documentation is rendered. Not let's have some fun to see how errors with your file are displayed. Go and break this and that in your document. You see not docs are not rendered and in diagnostics you see some more details"


otherwise we do not have tut 1 -> tut 2 -> tut 3 flow

Reply:
also `asyncapi: '2.7.0'` is risky, we might reach it april next year 😄 

maybe do `asyncapi: '1.0.0'`

Reply:
I'm not sure where to paste `asyncapi new --example=tutorial.yml --no-tty` in studio?

Reply:
![image](https://user-images.githubusercontent.com/74637789/203547408-d3f5bc7c-2b44-470b-984a-0a71f5206f81.png)
I see this in studio too, how useful will that be?

Reply:
this is something people would have to generate on local first, and copy/paste

although, we could also enable the same tutorial.yml in the Studio, when one clicks on 
<img width="240" alt="Screenshot 2022-11-24 at 12 04 39" src="https://user-images.githubusercontent.com/6995927/203768894-15a4d961-fb06-402b-82cb-6d7b56407486.png">

🤔

Reply:
Do you wanna have invalid AsyncAPI document as template in the Studio? 😄

Reply:
oh actually if that helps but that will be used only for this tutorial "educational purposes only" right?

Reply:
> It's not for me to decide

@magicmatatjahu kinda is as you are one of Studio maintainers 😆 

but yeah, that would be marked as Tutorial with description that it is for education with docs only

Reply:
Ok, should I create PR or @starlightknown? I can help.

Reply:
just folks remember, in studio we need current yaml 😄 

Just flow of this tutorial must indicate that user selects valid template and then just breaks the document in studio to simulate errors